addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -423,27 +423,25 @@
             <t t-set="can_access_channel" t-value="slide.channel_id.is_member or slide.channel_id.can_publish"/>
             <t t-if="can_access_channel">
                 <t t-set="links" t-value="slide.slide_resource_ids.filtered(lambda res: res.resource_type == 'url')"/>
-                <div class="col-12 col-md d-flex align-items-start mb-4 mb-md-0" t-if="links">
-                    <span class="text-muted fw-bold me-3">External sources</span>
-                    <div class="text-muted me-auto border-start ps-3">
+                <div class="d-flex mb-4 mt-4" t-if="links">
```

Comment:
If you're using "col" inside, shouldn't this div have "row" instead of "d-flex" ?